hack/util.sh

```diff
@@ -641,7 +641,11 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
+	local tmplsubsets='{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}'
+	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template='"'$tmplsubsets' $config"' || echo 0)" != "0" ]]' $((5*TIME_MIN))
+	local tmplready='{{ range .items }}{{ range .status.conditions }}{{ if eq .type "Ready" }}{{ .status }}{{ end }}{{ end }}{{ end }}'
+	wait_for_command "oc get pod -l deploymentconfig=docker-registry --output-version=v1 --template='$tmplready' $config | grep -qi '^True$'" $TIME_MIN
```

Comment:
For this one we should use `jsonpath`, and do we we need to be always looking at the latest condition?
```
os::cmd::wait_until_text "oc get pod -l deploymentconfig=docker-registry --jsonpath='{.status.conditions[?(@.type=="Ready")].status}'" "True"
```